src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/deprecated_parameter.json

```diff
@@ -0,0 +1,6 @@
+{
+    "deprecated_foo": {
+        "value": "bar",
+        "deprecated": "Since symfony\/framework-bundle 6.4: The parameter \"deprecated_foo\" is deprecated."
```

Comment:
let's move all depreactions under `_deprecations` to remove the BC break at the format level

Reply:
Done.